spec/unit/lib/metrics/volumetrics_spec.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+require_relative "./s3_fake_client"
+
+describe Metrics::Volumetrics do
+  let(:today) { Date.today }
+  let(:earlier_today) { today - 0.5 }
```

Comment:
Won't this end up being yesterday, if it's called during the morning?  If we just mean "within the last 24 hours" though (as opposed to midnight to midnight), that's fine.

Same question applies for spec/unit/lib/metrics/completion_rate_spec.rb

Reply:
Ah yes I see what you mean and that would be correct. I meant this to mean half a day ago. I'll see if I can rename this.